src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,34 +1,64 @@
 package edu.kis.vh.nursery;
 
+/**
+ * DefaultCountingOutRhymer is counting-out rhyme used to conducting complicated number games
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] numebrs = new int[12];
+    private static final int CAPACITY = 12;
+    private static final int STACK_EMPTY_VALUE = -1;
+    private static final int STACK_FULL_VALUE = 11;
 
-    public int total = -1;
+    private int[] NUMBERS = new int[CAPACITY];
 
-    public void countIn(int in) {
+    private int STACK_EMPTY_INDICATOR = STACK_EMPTY_VALUE; //STACK_EMPTY_INDICATOR wskazuje ostatni element

```

Comment:
This is not a constant (why STACK_EMPTY_INDICATOR uppercase? and why such name?)